tests/unit/s2n_aes_sha_composite_test.c

```diff
@@ -62,7 +62,7 @@ int main(int argc, char **argv)
     conn->client = &conn->initial;
 
     const int max_aligned_fragment = S2N_DEFAULT_FRAGMENT_LENGTH;
-    const uint8_t proto_versions[3] = { S2N_TLS10, S2N_TLS11, S2N_TLS12 };
+    const uint16_t proto_versions[3] = { S2N_TLS10, S2N_TLS11, S2N_TLS12 };
```

Comment:
Why is this correct but uint8_t was wrong?

They're #defines and not large: https://github.com/awslabs/s2n/blob/0a4dba65995c38ce0f4f4df8ab2654c54c562434/api/s2n.h#L43-L48
And we set uint8_t to them in plenty of other places: https://github.com/awslabs/s2n/blob/3c3ec85dd5d2d93b1f268404c8e7f74a0ef7b147/tls/s2n_tls.c#L21